최주현/5주차_02170.java

```diff
@@ -0,0 +1,56 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.StringTokenizer;
+
+public class Main {
+    static int n;
+    static int[][] drawLines;
+    static final int x = 0, y = 1;
```

Comment:
static final int~ 면 상수를 선언하신건데
일반적으로 상수는 대문자로 작성하니까 
```suggestion
    static final int X = 0, Y = 1;
```
이 별거 아니지만 더 좋아보이네요